StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -509,12 +509,17 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
             return
         }
 
+        guard let ephemeralKeySecret = configuration.customer?.ephemeralKeySecretBasedOn(intent: intent) else {
+            stpAssert(true, "Failed to read ephemeral key")
```

Comment:
Should we send an unexpected error analytic?

Reply:
Yeah probably, added one with some metadata.